### PR TITLE
Converting documents in Microsoft Azure (PDFNET-57973)

### PR DESCRIPTION
**Pull request by alevsh**
feat: initial upload

### CODE REVIEW

File: net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+
+In Visual Studio Code run:
+```bash
+dotnet restore
+```
+
+## Configure Aspose License
+In Visual Studio:
+1. Copy your Aspose.PDF license file to the project.
+2. Right-click on the license file.
```
Comment (ilyazhuykov) [inline net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md:119]: Add "Select Properties"

File: net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+            "stopAtEntry": false,
+            "env": {
+                "ASPNETCORE_ENVIRONMENT": "Development"
+            }
+        }
+    ]
+}
+```
+
+3. Create project structure:
```
Comment (ilyazhuykov) [inline net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md:97]: change 3 to 4

File: net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+3. Set "Copy to Output Directory" to "Copy always".
+4. Add license initialization code in Program.cs:
+   ```csharp
+   var license = new Aspose.Pdf.License();
+   license.SetLicense("Aspose.PDF.lic");
+   ```
+
+### Create code
+In Visual Studio:
+1. Right-click on Controllers folder
```
Comment (ilyazhuykov) [inline net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md:129]: Add step "Create folder "Controllers"" or change to "Create & right click"

File: net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+            }
+        }
+    ]
+}
+```
+
+3. Create project structure:
+```bash
+mkdir Controllers
+touch Controllers/PdfController.cs
```
Comment (ilyazhuykov) [inline net/other-programming-languages/microsoft-azure/converting-documents-azure-app-service/_index.md:100]: seems that touch is Linux specific command, probably we need to specify this

**Review by ilyazhuykov — APPROVED** [review]

**Review by ilyazhuykov — APPROVED** [review]
comments for offline part are fixed, approved

**Review by AnHolub — APPROVED** [review]

**Review by AlexBiriukov — APPROVED** [review]
No comments

**Pull request merged**